README.md

```diff
@@ -1,8 +1,251 @@
 # FuseML Core
 
-This repository contains the FuseML APIs definitions and core service.
+This repository contains the FuseML APIs definitions and core service. For the general information about FuseML project, check the [main repository page](https://github.com/fuseml/fuseml).
+
+## Installation
+
+* If you are using [fuseml installer](https://github.com/fuseml/fuseml), the core service is installed into your Kubernetes cluster along with other components and the command line client is downloaded to your working directory. It is recommended to copy the client to the location with other executables, e.g. `/usr/local/bin/` on the Linux systems.
+
+
+* To download pre-built client and server components, check the [releases page](https://github.com/fuseml/fuseml-core/releases).
+
+
+* To build the latest version directly from sources:
+
+  * Make sure you have `go` installed, at least version 1.16.
+
+  * Download [protocol buffers](https://github.com/protocolbuffers/protobuf). Use the release page for [version 3.15.7](https://github.com/protocolbuffers/protobuf/releases/tag/v3.15.7) and install only the `protoc` binary.
+
+    On Linux, you can proceed this way:
+    ```bash
+    wget https://github.com/protocolbuffers/protobuf/releases/download/v3.15.7/protoc-3.15.7-linux-x86_64.zip
+    unzip protoc-3.15.7-linux-x86_64.zip
+    sudo cp -a bin/protoc /usr/local/bin/
+    ```
+
+  * Clone `fuseml-core` repository
+    ```bash
+    git clone git@github.com:fuseml/fuseml-core.git
+    cd fuseml-core
+    ```
+
+  * Install go dependencies
+    ```bash
+    make deps
+    ```
+
+  * Generate server, client and CLI code and build the binaries.
+    ```bash
+    make all
+    ```
+    This will produce server (`bin/fuseml_core`) and command line client (`bin/fuseml_core-cli*`) binaries. The client binaries are by default compiled for multiple architectures, so pick one you need according to your OS. For convenience, you might create a link to the binary with more reasonable name, like
+
+    ```bash
+    ln -s bin/fuseml_core-cli-linux-amd64 bin/fuseml_core-cli
+    ```
+
+## Usage
+
+* Run the server localy
```

Comment:
I am not sure about offering the option to run `fuseml_core` outside of the environment provided by `fuseml-installer`. If we go that way we should also mention everything that is required to be able to run it properly (gitea, tekton, istio, ...), not only the environment variables.

IMO we should simple not offer that.

Reply:
This is not official documentation, this is a README file, which should serve developers too. I agree that we should omit such option from official documentation, but here it makes sense.

Reply:
I'll update the wording to reflect the situation